classes/content/sqlicontent.php

```diff
@@ -62,7 +62,7 @@ class SQLIContent
 
     /**
      * Fieldset Holder
-     * @var SQLIFieldsetHolder
+     * @var SQLILocationSet|SQLILocation[]
```

Comment:
This is what PHPStorm told me in the context of a current project when I stepped through the code. So to be honest, I am not 100% sure, but I trust PHPStorm :)